src/main/java/com/minecolonies/coremod/colony/jobs/JobChickenHerder.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.coremod.client.render.RenderBipedCitizen;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.herders.EntityAIWorkChickenHerder;
+import com.minecolonies.coremod.entity.ai.citizen.herders.EntityAIWorkCowboy;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.entity.ai.citizen.herders.EntityAIWorkCowboy'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)